ipa-core/src/protocol/ipa_prf/shuffle/sharded.rs

```diff
@@ -100,7 +104,7 @@ trait ShuffleContext: ShardedContext {
                         Direction::Right => r,
                     };
 
-                    item.borrow().clone() + mask
+                    item + mask
```

Comment:
👍

Reply:
This works here, but with the revised bounds in #1397 (which remove the implication that `ShuffleShare` is `Copy`), it comes back in a slightly different form.